packages/contracts-bedrock/scripts/DeployOPChain.s.sol

```diff
@@ -298,10 +307,66 @@ contract DeployOPChain is Script {
         return dso.output();
     }
 
-    function run(DeployOPChainInput _dsi, DeployOPChainOutput _dso) public view {
+    function run(DeployOPChainInput _dsi, DeployOPChainOutput _dso) public {
         require(_dsi.inputSet(), "DeployOPChain: input not set");
 
-        // TODO call OP Stack Manager deploy method
+        OPStackManager opsm = _dsi.opsm();
+
+        OPStackManager.Roles memory roles = OPStackManager.Roles({
+            opChainProxyAdminOwner: _dsi.opChainProxyAdminOwner(),
+            systemConfigOwner: _dsi.systemConfigOwner(),
+            batcher: _dsi.batcher(),
+            unsafeBlockSigner: _dsi.unsafeBlockSigner(),
+            proposer: _dsi.proposer(),
+            challenger: _dsi.challenger()
+        });
+        OPStackManager.DeployInput memory deployInput = OPStackManager.DeployInput({
+            roles: roles,
+            basefeeScalar: _dsi.basefeeScalar(),
+            blobBasefeeScalar: _dsi.blobBaseFeeScalar(),
+            l2ChainId: _dsi.l2ChainId()
+        });
+
+        vm.broadcast(msg.sender);
+        OPStackManager.DeployOutput memory deployOutput = opsm.deploy(deployInput);
```

Comment:
Good idea, we can strengthen these getter methods to check that (1) a proxy is at the address, and (2) it has an implementation set. Let's track this in an issue to avoid making this PR too big

Reply:
Made an issue, I created a new `opsm` label for issue tracking: https://github.com/ethereum-optimism/optimism/issues/11814 

Not sure if you've already got a system in place.

Reply:
Perfect, thank you! I've also linked to that issue from a figjam sticky, which we'll keep using for project tracking until yoon migrates us to the new system